server/vmap/trillian_map_server/main.go

```diff
@@ -132,15 +115,9 @@ func main() {
 
 	// Bring up the RPC server and then block until we get a signal to stop
 	rpcServer := startRPCServer(lis, *serverPortFlag, registry)
-	go awaitSignal(rpcServer)
+	defer glog.Flush()
```

Comment:
This will not work the way you want I fear... `glog.Fatalf` below will do *stuff* and then call `os.Exit`, but the [docs for os.Exit](https://golang.org/pkg/os/#Exit) have this to say: _"... deferred functions are *not* run"_.